tests/keras_tests/feature_networks_tests/test_features_runner.py

```diff
@@ -546,10 +546,6 @@ def test_gptq(self):
                         rounding_type=RoundingType.SoftQuantizer,
                         quantizer_config=SoftQuantizerConfig(),
                         per_channel=True).run_test()
-        GradientPTQTest(self,
-                        rounding_type=RoundingType.SoftQuantizer,
-                        quantizer_config=SoftQuantizerConfig(),
-                        per_channel=True).run_test(experimental_facade=True)
```

Comment:
Why this test has been removed?

Reply:
it's a duplicate test. Enough to run without the experimantal_facase parameter, as itno longer exists